src/handler.js

```diff
@@ -46,11 +45,25 @@ const responseWithEmptyBody = () => {
   return response
 }
 
-const isInvalidValidEvent = (event) => !('path' in event && 'httpMethod' in event)
+const isInvalidValidEvent = (event) => !(hasPath(event) && 'httpMethod' in event)
```

Comment:
InvalidValid?

Reply:
Named so can cover any outcome.... Renamed to something less dualistic -> isInvalidEvent